scripts/plots/aqm/exevs_aqm_grid2obs_plots.sh

```diff
@@ -58,7 +59,7 @@ for aqmtyp in ozone pm25 ozmax8 pmave; do
             DAY=`cut -c 1-8 curdate`
             cpfile=evs.stats.${COMPONENT}_${biasc}.${RUN}.${VERIF_CASE}_${aqmtyp}.v${DAY}.stat
             sedfile=evs.stats.${aqmtyp}_${biasc}.${RUN}.${VERIF_CASE}.v${DAY}.stat
-            if [ -e ${EVSINaqm}.${DAY}/${cpfile} ]; then
+            if [ -s ${EVSINaqm}.${DAY}/${cpfile} ]; then
                 cpreq ${EVSINaqm}.${DAY}/${cpfile} ${STATDIR}
```

Comment:
@Ho-ChunHuang-NOAA Should this cpreq be changed to cp -v as well?

@PerryShafran-NOAA

Reply:
@AliciaBentley-NOAA @PerryShafran-NOAA This is the stats copying back from COMOUT to DATA for plotting, so it is "input file" not output file for plots steps.

My understanding from Wei's comment 'cp -v' is for files to be copied from DATA to COMOUT

Reply:
@PerryShafran-NOAA I am following Alicia's instruction on (2)

"1) add file checks before cpreq is used to copy files from /tmp to /com, and 2) change cpreq to cp when moving files from /tmp to /com."

Wei's comment indicates that critical input file should use cpreq.

Reply:
@Ho-ChunHuang-NOAA We will leave this as is.